src/lib/jar.js

```diff
@@ -80,6 +81,8 @@ var Jar = {
             if (err) return callback(err);
             Jar.unsign(value, function (err, contents) {
                 if (err) return callback(err);
+                if (typeof contents === 'undefined') return callback(null, contents);
```

Comment:
We could also avoid this if we handled the empty value ahead of `Jar.unsign`. Not a big deal, but would simplify what happens a little bit. "get —> check undefined —> return" instead of "get —> unsign —> check undefined —> return —> check undefined —> return".